puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_develop.yaml

```diff
@@ -0,0 +1,50 @@
+- job:
+    name: test_develop
+    project-type: matrix
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 32
+    properties:
+      - github:
+          url: https://github.com/theforeman/foreman
+    scm:
+      - foreman
+    triggers:
+      - github
+      - pollscm: '*/5 * * * *'
+      - schedule-failed-builds: '* * * * *'
+    axes:
```

Comment:
There's a missing slave axis, with label "fast".  (These are slaves with SSDs etc., use them for core Foreman tests, especially integration tests.)